src/ContainerCommitModal.jsx

```diff
@@ -93,8 +93,6 @@ class ContainerCommitModal extends React.Component {
 
         varlink.call(utils.PODMAN_ADDRESS, "io.podman.Commit", commitData)
                 .then(() => {
-                    this.props.updateImagesAfterEvent();
-                    this.props.updateContainersAfterEvent();
```

Comment:
Could now be simplified to just `then(this.props.onHide)`

Reply:
done